edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -315,21 +361,19 @@ def _decode_and_verify_token(token, jwt_issuer):
     return decoded_token
 
 
-def _get_signing_jwk_key_set(jwt_issuer, add_symmetric_keys=True):
+def get_verification_jwk_key_set(asymmetric_keys=None, secret_key=None):
     """
     Returns a JWK Keyset containing all active keys that are configured
     for verifying signatures.
```

Comment:
Docstring should be updated -- currently describes reading from configuration. Maybe something like this:

```suggestion
    Creates a JWK Keyset containing the provided keys.
    
    Args:
        asymmetric_keys (list or None): List of asymmetric JWK verification keys,
            each in JSON format.
        secret_key (str or None): Secret key for symmetric JWT verification, as an
            unencoded string.
```

Reply:
done.